src/CMakeLists.txt

```diff
@@ -48,3 +48,12 @@ install(EXPORT libappimageTargets
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/libappimage
     COMPONENT libappimage-dev
 )
+
+# Pkg Config
+CONFIGURE_FILE("${PROJECT_SOURCE_DIR}/cmake/libappimage.pc.in" "${PROJECT_BINARY_DIR}/libappimage.pc" @ONLY)
+
+install(FILES
+    "${PROJECT_BINARY_DIR}/libappimage.pc"
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/pkgconfig
```

Comment:
The install dir seems not correct to me, should that not be `share/pkgconfig` or even `${CMAKE_INSTALL_LIBDIR}/pkgconfig`? At least I have not seen a direct `pkgconfig/` subdir on my system in the usual prefixes.

Also is usually the `${CMAKE_INSTALL_PREFIX}` not passed, `install() will add that itself to relative paths, and then it also seems not recommended to allow the use of DESTDIR at install time. See also https://cmake.org/cmake/help/v3.0/command/install.html

Reply:
This one here left me unsatisfied, as I could not find a hint on https://www.freedesktop.org/wiki/Software/pkg-config/ where actually one should install the pc files. Only found hints on 3rd-party pages or what is implemented in https://cgit.kde.org/extra-cmake-modules.git/tree/modules/ECMGeneratePkgConfigFile.cmake or what is and works on my system.

Asked some openSUSE TW packagers for a start, and they hinted that "/usr/share/pkgconfig shall be used for arch-independent stuff".
So `${CMAKE_INSTALL_LIBDIR}/pkgconfig` seems the right thing here (with that var `CMAKE_INSTALL_LIBDIR` set by using cmake's GnuInstallDirs, as done already)

Reply:
Agreed, done.